x-pack/plugins/maps/public/classes/layers/wizards/load_layer_wizards.ts

```diff
@@ -29,28 +29,33 @@ import { choroplethLayerWizardConfig } from './choropleth_layer_wizard';
 import { newVectorLayerWizardConfig } from './new_vector_layer_wizard';
 
 let registered = false;
+export const layersForRegistration = [
```

Comment:
I use this layer list in `add_layer_panel/view.tsx` to pass the selected layer with its full configuration to the `_onWizardSelect` method, which opens the selected layer wizard.

Reply:
This is going to cause problems because layer wizards can be added by other plugins. Instead, add a new method to `layer_wizard_registry` to find a wizard by id and use that method in `add_layer_panel/view.tsx`.